### ISSUE TITLE
Open Search: Remove duplicate navigation landmark

### ISSUE THREAD

**Issue by alexstine** [WordPress/wporg-mu-plugins#311]
Currently, the Open Search button is placed in a dedicated navigation landmark. One button does not need a dedicated landmark. This adds unnecessary clutter to the landmark structure and should be made apart of the main nav.

Responsible code:
```
<nav class="is-responsive  is-vertical global-header__search wp-block-navigation is-vertical is-layout-flex wp-container-6" aria-label="">
```

Please note, this comment:
```
<!--
		The search block is inside a navigation menu because that provides the exact functionality the design
		calls for. It also provides a consistent experience with the primary navigation menu, with respect to
		keyboard navigation, ARIA states, etc. It also saves having to write custom code for all the interactions.
	-->
```

Not sure what to do with this since it seems like navigation block is giving us accessibility support. This would be considered a bug and I think it likely came out of a previous A11Y implementation. I likely missed it during a review. Not a high priority issue but something that should be fixed shortly to avoid clutter for users.

**Comment by alexstine** [comment]
CC @ryelle. I believe you worked on this last.

**Comment by ryelle** [comment]
> it seems like navigation block is giving us accessibility support. 

Yeah, basically. The navigation block handles the whole interaction pattern, visual and accessibility. The search field is in a dropdown, toggled open and closed with that Open Search button. It's in a nav block on its own so that it can be toggled independently from the main navigation.

I don't think it's a simple case of switching out the HTML element, because that would invalidate the block and the navigation JS we're using might stop working.

Maybe the button & dropdown could be built in a nav submenu instead, so we could add it to the end of the main navigation. But that wouldn't work on mobile, where they are intentionally two different overlays. Hmm, this is trickier than it sounds. I agree it should be cleaned up, though.

> This adds unnecessary clutter to the landmark structure and should be made apart of the main nav.

Would it be okay if the button was just not in its own landmark anymore, or does it need to be part of the main nav?

**Comment by alexstine** [comment]
I think it would be better if it were apart of the main nav. Voiceover on Mac for example, intentionally marks the end of landmarks with "End of navigation landmark" type text.
